File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/d/SkinTestUsePharmRstDTO.java
package iih.ci.ord.dto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;

import java.math.BigDecimal;

/**
 * 患者皮试或用药相关数据结果信息DTO DTO数据 
 * 
 */
public class SkinTestUsePharmRstDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * N天有效返回结果枚举值
	 * @return Integer
	 */
	public Integer getNdaysvalidrst() {
		return ((Integer) getAttrVal("Ndaysvalidrst"));
	}
	/**
	 * N天有效返回结果枚举值
	 * @param Ndaysvalidrst
	 */
	public void setNdaysvalidrst(Integer Ndaysvalidrst) {
		setAttrVal("Ndaysvalidrst", Ndaysvalidrst);
	}
	/**
	 * 皮试医嘱主键标识
	 * @return String
	 */
	public String getId_orskin() {
		return ((String) getAttrVal("Id_orskin"));
	}
	/**
	 * 皮试医嘱主键标识
	 * @param Id_orskin
	 */
	public void setId_orskin(String Id_orskin) {
		setAttrVal("Id_orskin", Id_orskin);
	}
	/**
	 * 过敏日期
	 * @return
	 */
	public FDateTime getDt_act()
	{
		return ((FDateTime) getAttrVal("Dt_act"));
	}
	/**
	 * 过敏日期
	 * @param Dt_act
	 */
	public void setDt_act(FDateTime Dt_act) {
		setAttrVal("Dt_act", Dt_act);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/d/SkinTestUsePharmRstDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.d
{
    /// <summary>
    /// SkinTestUsePharmRstDTO 的摘要说明。
    /// </summary>
    public class SkinTestUsePharmRstDTO : BaseDTO {

        public SkinTestUsePharmRstDTO() {
 
        }

        /// <summary>
        /// N天有效返回结果枚举值
        /// </summary>
		public int? Ndaysvalidrst {
            get { return getAttrVal<int?>("Ndaysvalidrst",null); }
            set { setAttrVal<int?>("Ndaysvalidrst", value); }
        }

        /// <summary>
        /// 皮试医嘱主键标识
        /// </summary>
		public string Id_orskin {
            get { return getAttrVal<string>("Id_orskin",null); }
            set { setAttrVal<string>("Id_orskin", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Ndaysvalidrst", "Id_orskin"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.d.SkinTestUsePharmRstDTO";
        }
    }
}
